pkg/kubelet/dockertools/manager.go

```diff
@@ -579,7 +579,7 @@ func (dm *DockerManager) runContainer(
 		}
 	}
 	memoryLimit := container.Resources.Limits.Memory().Value()
-	cpuShares := milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
+	cpuShares := milliCPUToShares(container.Resources.Requests.Cpu().MilliValue())
```

Comment:
Am I missing something, or did this PR just entirely drop support for ```container.Resources.Limits``` on CPU?

That would have some bad effects for us in OpenShift without preserving support for Limits in some sensible fashion.  Still reviewing rest of PR to see if I missed something.